project/threads/templates/threads/static_templates/landing.html

```diff
@@ -134,26 +129,26 @@ <h5 class="content-subtitle">Citizen/Representative Engagement</h5>
     <div class="lastcall cushion">
         <div class="container section">
             <div class="section">
-                <h5 class="center text-call">See what your voice can bring to your community</h5>
+                <h5 class="center text-call">{% trans "See what your voice can bring to your community" %}</h5>
             </div>
             <div class="section row center">
-                <a href="login" class="btn btn-large waves-effect waves-light"><span class="dark-grey-text">Join CiviWiki</span></a>
+                <a href="login" class="btn btn-large waves-effect waves-light"><span class="dark-grey-text">{% trans "Join" %}  CiviWiki</span></a>
```

Comment:
Full sentence for translators 
```suggestion
                <a href="login" class="btn btn-large waves-effect waves-light"><span class="dark-grey-text">{% trans "Join CiviWiki" %} </span></a>
```